stages/org.osbuild.zip.meta.json

```diff
@@ -0,0 +1,50 @@
+{
+  "summary": "Assembles the tree into a zip archive named `filename`",
+  "description": [
+    "Buildhost commands used: `zip`"
+  ],
+  "capabilities": [
+    "CAP_MAC_ADMIN"
```

Comment:
Hmm, I don't think you need CAP_MAC_ADMIN for this stage.

Reply:
this may be copied from `org.osbuild.tar`, does `tar` really need it?

Reply:
It has been added for `org.osbuild.tar` stage and few others by design here: https://github.com/osbuild/osbuild/commit/ad5638bf14adad8ce0be3f8ee4017f70193bc083

Reply:
There's an important difference between zip and tar, though. The tar stage needs it, because the stage can optionally preserve xattrs when creating a tar archive, and CAP_MAC_ADMIN is required for reading selinux labels (encoded as xattrs) unknown to the host (see https://github.com/osbuild/osbuild/commit/ad5638bf14adad8ce0be3f8ee4017f70193bc083). However, zip doesn't care at all about xattrs, so I believe that CAP_MAC_ADMIN is indeed useless here.